activerecord/CHANGELOG.md

```diff
@@ -1,4 +1,13 @@
-## Rails 7.1.0.beta1 (September 13, 2023) ##
```

Comment:
This should be kept and the new entry should go above it (with a newline in between)

Reply:
Ah. I was confused as I didn't see this comment in the source in `main` so I made a guess. I'll add it back, rebase, and relish shortly

Reply:
@skipkayhil Fixed.